patches/PR5/autosprintf.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/gettext-runtime/libasprintf/autosprintf.cc b/gettext-runtime/libasprintf/autosprintf.cc
+index 429b8e8..2c6671f 100644
+--- a/gettext-runtime/libasprintf/autosprintf.cc
++++ b/gettext-runtime/libasprintf/autosprintf.cc
+@@ -43,7 +43,7 @@ namespace gnu
+   {
+     va_list args;
+     va_start (args, format);
+-    if (vasprintf (&str, format, args) < 0)
++    if (std::vasprintf (&str, format, args) < 0)
```

Comment:
Interesting, since zoslib is implemented in C++, we could probably expose a C vasprintf that calls std::vasprintf.